.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,10 @@
+---
+name: Bug report
+about: To report a part of mocha not working as expected
+title: ''
+labels: 'unconfirmed-bug'
+---
+
 <!--
 Have you read Mocha's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md
 For more, check out the Mocha Gitter chat room: https://gitter.im/mochajs/mocha
```

Comment:
| We recommend avoiding the use of globally installed Mocha.

We recommend that you _not_ install Mocha globally.

Reply:
Not completely addressed. Make "not" bold or emphatic...

Reply:
Still not addressed...